localesetup/localesetup

```diff
@@ -79,7 +91,7 @@ locale="$(eval $dialog \
 --cancel-label "`eval_gettext 'Exit'`" \
 --menu \
 \"\\n`eval_gettext 'The current locale is $currentlocale.'` \\n\\n\
-`eval_gettext 'Please select your preferred system language:'`\" 0 0 11 \
+`eval_gettext 'Please select your preferred system language:'`\" 0 0 0 \
```

Comment:
I actually like the 11 height better. Leaving it to become as large as the screen makes the menu intimidating. That was the reason why we had it restricted to 11 in the first place. Scrolling works fine in any case.